Domain Services/Health/Expense-v1.1.0.yml

```
@@ -0,0 +1,654 @@
+          type: string
+          minLength: 2
+          maxLength: 64
+        ids:
+          type: array
+          description: Search by the payment ids 
+          format: uuid
+          items:
+            type: string
+            example: 44452d1f-d88b-4dee-b87d-679756a88161
+        billIds:
+          description: search payment by the bill ids         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)